src/bors/handlers/info.rs

```diff
@@ -0,0 +1,58 @@
+use crate::bors::Comment;
+use crate::bors::RepositoryState;
+use crate::database::PgDbClient;
+use crate::github::PullRequest;
+use std::sync::Arc;
+
+pub(super) async fn command_info(
+    repo: Arc<RepositoryState>,
+    pr: &PullRequest,
+    db: Arc<PgDbClient>,
+) -> anyhow::Result<()> {
+    // Geting PR info from database
+    let pr_model = db.get_pull_request(repo.repository(), pr.number).await?;
+
+    // Building the info message
+    let mut info_lines = Vec::new();
+
+    // Approval info
+    if let Some(pr) = pr_model {
```

Comment:
Let's use `get_or_create_pull_request`, there's no point in posting that the PR was "not found" when it clearly exists :)